### ISSUE TITLE
Heat recovery heat exchanger is sized to two different design supply air flow rates when used Zone ERV object

### ISSUE THREAD

**Issue by Nigusse** [NREL/EnergyPlus#5773]
Heat recovery heat exchanger in ZoneHVAC:EnergyRecoveryVentilator object is auto sized in the parent object and as a child object in heat recovery module. 

The heat exchanger design supply air flow rate in ZoneHVAC ERV is sized to a design outdoor air or ventilation air flow requirement while in heat recovery model it is sized to the zone design flow rate. And both autosized values are reported to the eio file and are different.  

When heat exchanger is used in Zone HVAC ERV object it should be sized to the ventilation requirement flow not to the design air flow rate.

Added a defect file:
## Issue overview

Please change this line to a description of the defect, with useful supporting information including how the issue was found, conditions that trigger this defect, workarounds for users of the current version, and a clear description of how to validate that a fix is complete for this defect.
### Details

Some additional details for this issue (if relevant):
- Platform (Operating system, version)
- Version of EnergyPlus (if using an intermediate build, include SHA)
- Unmethours link or helpdesk ticket number
### Checklist

Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
- [ ] Defect file added (list location of defect file here)
- [ ] Ticket added to Pivotal for defect (development team task)
- [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by mjwitte** [comment]
@Nigusse Hmm, I thought this was all straightened out here in #4668.

**Comment by Nigusse** [comment]
@mjwitte It must have been missed. I have added EIO file as part of the defect file for quick look.

**Comment by mjwitte** [comment]
@Nigusse It's not sizing to the design supply flow, but it's initially sizing to the ventilation rate specified in the zone sizing object which isn't consistent with the ventilation rate specified in the ERV object.  So, what it's currently reporting for design size is correct - it's the sizing ventilation rate.  But it shouldn't be reporting HX size three times.  And we should consider changing the ERV to autosize off the normal sizing inputs instead of having its own ventilation rate inputs.  @rraustad

**Comment by mjwitte** [comment]
@Nigusse Strike that last comment.  (deleted above).  But I went back to the test file from the earlier fix which was implemented in v8.3.  It differs from this defect file in two ways - it didn't have sizing:zone object and the simulation control sizing flags were set to No.  So, I made it equivalent to this one and ran it with v8.3 and v8.4.  Something changed in v8.4 to add the extra sizing outputs.  Not sure where that change happened.

```
v8.3 eio
 Component Sizing Information, ZoneHVAC:EnergyRecoveryVentilator, STAND ALONE ERV 1, Design Size Supply Air Flow Rate [m3/s], 7.78779E-002
 Component Sizing Information, HeatExchanger:AirToAir:SensibleAndLatent, OA HEAT RECOVERY 1, Design Size Nominal Supply Air Flow Rate [m3/s], 7.78779E-002
 Component Sizing Information, ZoneHVAC:EnergyRecoveryVentilator, STAND ALONE ERV 1, Design Size Exhaust Air Flow Rate [m3/s], 7.78779E-002
 Component Sizing Information, Fan:OnOff, STAND ALONE ERV SUPPLY FAN, Design Size Maximum Flow Rate [m3/s], 9.34535E-002
 Component Sizing Information, Fan:OnOff, STAND ALONE ERV EXHAUST FAN, Design Size Maximum Flow Rate [m3/s], 9.34535E-002

v8.4 eio
 Component Sizing Information, ZoneHVAC:EnergyRecoveryVentilator, STAND ALONE ERV 1, Design Size Supply Air Flow Rate [m3/s], 7.78779E-002
 Component Sizing Information, HeatExchanger:AirToAir:SensibleAndLatent, OA HEAT RECOVERY 1, Design Size Nominal Supply Air Flow Rate [m3/s], 7.78779E-002
 Component Sizing Information, ZoneHVAC:EnergyRecoveryVentilator, STAND ALONE ERV 1, Design Size Exhaust Air Flow Rate [m3/s], 7.78779E-002
 Component Sizing Information, HeatExchanger:AirToAir:SensibleAndLatent, OA HEAT RECOVERY 1, Design Size Nominal Supply Air Flow Rate [m3/s], 0.47892
 Component Sizing Information, HeatExchanger:AirToAir:SensibleAndLatent, OA HEAT RECOVERY 1, User-Specified Nominal Supply Air Flow Rate [m3/s], 7.78779E-002
 Component Sizing Information, Fan:OnOff, STAND ALONE ERV SUPPLY FAN, Design Size Maximum Flow Rate [m3/s], 9.34535E-002
 Component Sizing Information, Fan:OnOff, STAND ALONE ERV EXHAUST FAN, Design Size Maximum Flow Rate [m3/s], 9.34535E-002
```

**Comment by Nigusse** [comment]
@mjwitte Looking at the results for v8.4 above, the HX is sized again in the HeatRecovery module, I think.

**Comment by mjwitte** [comment]
Closed via #5779
